code.json

```diff
@@ -0,0 +1,48 @@
+[
+  {
+    "name": "geoknife",
+    "organization": "U.S. Geological Survey",
+    "description": "Processes gridded datasets found on the U.S. Geological Survey Geo Data Portal web application or elsewhere, using a web-enabled workflow that eliminates the need to download and store large datasets that are reliably hosted on the Internet. The package provides access to several data subset and summarization algorithms that are available on remote web processing servers.",
+    "version": "1.6.1",
+    "status": "Production",
+    
+    "permissions": {
+      "usageType": "openSource",
+      "licenses": [
+        {
+          "name": "Public Domain, CC0-1.0",
+          "URL": "https://code.usgs.gov/WMA-Data-Science/dataRetrieval/blob/master/LICENSE.note"
+        }
+        ]
+    },
+    
+    "homepageURL": "https://github.com/USGS-R/geoknife",
```

Comment:
You'll want to change this to the code.usgs.gov address. That's the "OFFICIAL HOME"